core/fmt/fmt.odin

```diff
@@ -2405,6 +2406,21 @@ fmt_named :: proc(fi: ^Info, v: any, verb: rune, info: runtime.Type_Info_Named)
 			write_padded_number(fi, (ns), 9)
 			io.write_string(fi.writer, " +0000 UTC", &fi.n)
 			return
+
+		case regex.Regular_Expression:
```

Comment:
Is it a good idea? My gut is unsure about this, and whether the need to print a regular expression value is necessary or not.

Reply:
Compromise suggestion? Make it a helper proc in `core:text/regex` that takes a string builder.

Reply:
I can see the reason in removing the print ability entirely, because the `original_pattern` value depends on an external lifetime. I.e. if someone creates a RegEx value, deletes the string that created it, then tries to print it, they'll have problems, but if someone makes a clone of the string, then they're responsible for deleting it along with the RegEx value, and that could complicate certain situations. Admittedly, it is a bit of a weak spot in the package because of that very dependency. I didn't want to introduce unneeded cloning of strings just to break it, either.

Maybe it is better to remove it? Are there any good uses for printing the actual RegEx value, outside of debugging? The only time I can imagine needing to print it myself would be if I had an editor for patterns in a larger application, and in that case, I would just manage and show the original string the user inputs instead.

Reply:
Sounds like it's better to take it, indeed.